hplsql/src/test/java/org/apache/hive/hplsql/TestHplsqlOffline.java

```diff
@@ -31,9 +27,7 @@
  */
 public class TestHplsqlOffline {
 
-  private final ByteArrayOutputStream out = new ByteArrayOutputStream();
-
-  @Test
+    @Test
```

Comment:
Please fix indentation.

Reply:
Fixed in https://github.com/apache/hive/pull/5381/commits/e96c827734d75bb87dc7be3354e63a2a285adefe